todos/src/components/Todos.js

```diff
@@ -32,14 +39,14 @@ function Todos() {
               completedDate={elem.completedDate}
               onEdit={elem.onEdit}
             />
-          );
+          ) : null;
         })}
       </div>
       {list.length === 0 && !isAddTaskVisible && <EmptyTaskList />}
-      {currentPage * PER_PAGE < list.length ? (
-        <LoadMoreBtn type={"Load More"} />
+      {currentPage * PER_PAGE < filteredTodos.length ? (
```

Comment:
Can we port this to a computed variable to make the markup less complex?

Reply:
Yes, @hhu-wtag Bhai. We've already done that on our pagination branch, so I resolved the conflicts, and now all the markup in both branches is correct, with less complexity. Thank you, Bhai for pointing it out again.